Scripts/script-ParsePCAP.yml

```diff
@@ -0,0 +1,294 @@
+
+        for info in self.get_http_request_data():
+            tmp = {}
+            for key, value in info.items():
+                tmp[key] = value
+            self._http_json.append(tmp)
+
+        for flow in self.get_flows():
+            direction = flow.split("=>")
+            source = direction[0].split("/")
+            destination = direction[1].split("/")
+            self._flow_json.append({"source_ip":source[0],"source_port":source[1],"destination_ip":destination[0],"destination_port":destination[1]})
+
+        #construct the output object
+        obj = {"file":self._pcap_file,
+               "dns_data":self._dns_json,
+               "destination_ip_details":self._dest_json,
+               "http_requests":self._http_json,
+               "flows":self._flow_json,
+               "counts":self._counts_json
+               }
+
+        return json.dumps(obj)
+
+
+
+filePath = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryId'] })
+readyPath = filePath[0]["Contents"]["path"]
+#demisto.log(filePath)
+#file = open(readyPath, 'rb')
+#data=file.read()
+#demisto.log(readyPath)
+#demisto.results( { "Type" : entryTypes["note"], "ContentsFormat" : formats["text"], "Contents" : "Testing" } )
+ret = []
+filename = filePath[0]["Contents"]["name"]
+demisto.log('PCAP FILE PROCESSED')
+demisto.log(filename)
+
+miner = pcap_miner(readyPath)
+jsonResults = miner.summary2json()
+pyResults = json.loads(jsonResults)
+#demisto.log(jsonResults);
+#demisto.log(pyResults);
+if(pyResults["counts"]):
+    displayData = tableToMarkdown('PCAP Data Frequency Counts', pyResults["counts"])
+if(pyResults["destination_ip_details"]):
+    displayData += tableToMarkdown('Destination IP Details', pyResults["destination_ip_details"])
+if(pyResults["dns_data"]):
+    displayData += tableToMarkdown('DNS Details', pyResults["dns_data"])
+if(pyResults["http_requests"]):
+    displayData += tableToMarkdown('Http Requests', pyResults["http_requests"])
+if(pyResults["flows"]):
+    displayData += tableToMarkdown('Flow Data', pyResults["flows"])
+
+demisto.results({'Type': entryTypes['note'], 'Contents': pyResults, 'EntryContext': {'pcap_results': pyResults}, 'ContentsFormat': formats['json'], 'HumanReadable': displayData})
+
+
+
+type: python
+tags: []
+comment: The ParsePCAP Automation Script is used to parse PCAP files. It currently
+  does not support PCAPNG files. So please convert your PCAPNG to a regular PCAP file.
+  There are several tools that are capable of doing this online.  If you have a PCAPNG
+  file and you would like to convert it please look at using editcap. For instance
+  if you are on a mac the default of tcpdump will be a PCAPNG file. To work around
+  this you can simply do the following tcpdump -w test.pcap -i en0 by making sure
+  that the interface is selected.
+enabled: true
+args:
+- name: entryId
+  default: true
+  description: Enter the entryId
```

Comment:
There should be an "outputs" section, where you declare the outputs of the scripts, so you can use those in a playbook (for next tasks)